test/src/e2e/account.test.ts

```diff
@@ -95,107 +94,100 @@ describe("account", function() {
                     message,
                     secret
                 );
+
```

Comment:
You commit contains unrelated changes. Please be careful when you create a commit.